### PR TITLE
Add OpenAI API call with node.js

### PR DESCRIPTION
**Pull request by molxno**
### Milestone
Fix OpenAI API to use .env and node.js library.

### CODE REVIEW

File: src/pages/content/hooks/useGpt.tsx [hunk 1]
```diff
@@ -1,45 +1,49 @@
...
-  const send = (text: string, mode: string) => {
-    setContext((ctx) => {
-      const NEW_CONTEXT = [...ctx, { role: "user", content: text }]
-      const PROMT = MODE_MAP_PROMNT[mode]
+    const send = (text: string, mode: string) => {
+        setContext((ctx) => {
+            const NEW_CONTEXT: ChatContextType[] = [...ctx, {role: "user", content: text}]
+            console.log("Mode:", mode);
+            const PROMPT = MODE_MAP_PROMPT[mode]
+            console.log("Mapped prompt:", PROMPT);
```
Comment (SrRiuz) [inline src/pages/content/hooks/useGpt.tsx:27]: Remove "console.log"

File: src/pages/content/components/FloatingMenu/Menu.tsx [hunk 2]
```diff
@@ -1,48 +1,50 @@
...
+                setSelectedOption(() => option)
+            }}/>
+            <IaButton
+                onClick={() => {
+                    if (selectedOption) {
+                        chrome.runtime.sendMessage({action: "openPopup"});
+                        setTimeout(() => {
+                            chrome.runtime.sendMessage({
+                                context: {
+                                    text: text,
```
Comment (SrRiuz) [inline src/pages/content/components/FloatingMenu/Menu.tsx:35]: Change indentation to 2

**Review by SrRiuz — CHANGES_REQUESTED** [review]

**Review by SrRiuz — COMMENTED** [review]
Remove the "gtp_api_key" property in the settings file

File: src/pages/content/components/FloatingMenu/Menu.tsx (hunk 2 above)
Comment (molxno) [reply src/pages/content/components/FloatingMenu/Menu.tsx:35]: We do this when choosing the code formatter, so we fix this kind of details.

**Pull request merged**